src/natcap/invest/urban_cooling_model.py

```diff
@@ -177,7 +177,7 @@
             },
             "about": (
                 "A CSV table containing information on energy consumption "
-                "for various types of buildings, in kW/degC."
+                "for various types of buildings, in kWh/degC/m^2."

```

Comment:
Should there be a space between `deg` and `C`? I think you've written it that way until now.

Reply:
Yes!  Way better to err on the side of consistency.  Patched in 686d91a8.